pallas-applying/Cargo.toml

```diff
@@ -13,8 +13,14 @@ authors = ["Maico Leberle <maico.leberle@gmail.com>"]
 doctest = false
 
 [dependencies]
+cryptoxide = "0.1"
```

Comment:
crypto algos should be made available through `pallas::crypto`, which already depends on `cryptoxide`.

Reply:
Just added a [commit](https://github.com/txpipe/pallas/commit/d74b1484e91b8c417a59442cae9b3d96caafbd81) correcting this.